app/src/main/java/com/mstudio/superheroarch/Character.kt

```diff
@@ -16,11 +21,12 @@ data class Ubication(
     val url: String
 ) : Serializable
 
+@Entity(tableName = "episodes")
```

Comment:
this table is not needed, just show an error if the episode details cannot be loaded.

Reply:
Fixed in next commit